documentation/src/main/asciidoc/topics/proc_displaying-metrics-st-operation.adoc

```diff
@@ -0,0 +1,41 @@
+[id='displaying-metrics-st-operationadoc_{context}']
+= Displaying metrics during a state transfer operation
+
+You can export time metrics for clustered caches that {brandname} redistributes across nodes. 
+
+A state transfer operation occurs when a clustered cache topology changes, such as a node joining a cluster, so the cache's data must be redistributed to all nodes in a cluster. 
+
+[NOTE]
+====
+You cannot perform a state transfer operation in invalidation mode, because this mode restricts database write operations after state changes occur. 
```

Comment:
Although it is correct that we do not execute state transfer for caches in invalidation mode, I think that the cause may be off. I might be wrong, but I think it does not execute the state transfer because of the design/use-case of the mode. If someone has another suggestion/explanation, otherwise, we can stick with only saying that invalidation does not execute a state transfer.

Reply:
Updated.